examples/SLAM/js/common/math/plane.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Helper class for maths regarding 3D Plane
+ * Striped down and refactored to use gl-matrix from https://github.com/mrdoob/three.js/blob/dev/src/math/Plane.js
+ */
+
+import { vec3 } from 'gl-matrix';
+
+
+class Plane {
+  public readonly isPlane: boolean = true;
+  public normal: vec3;
+  public constant = 0;
+
+	constructor( normal = vec3.fromValues( 1, 0, 0 ), constant = 0 ) {
+		this.normal = normal;
+		this.constant = constant;
+	}
+
+	set( normal: vec3, constant: number ) {
+
+    this.normal = vec3.clone(normal)
```

Comment:
Formatting is a little off here.

[This](https://github.com/WonderlandEngine/components/blob/main/.prettierrc.json) is our official prettier config, that plus `npm i -D --exact prettier && prettier './**/*.{js,ts}' --write` should get the code formatted nicely 😌